test/Api/CatalogApiTest.php

```diff
@@ -264,10 +54,10 @@ protected function deleteTestCatalog() {
      *
      */
     public function test_batchDeleteCatalogObjects() {
-      $coffeeId = $this->idMap[self::COFFEE_CLIENT_ID];
-      $smallCoffeeId = $this->idMap[self::SMALL_COFFEE_CLIENT_ID];
-      $largeCoffeeId = $this->idMap[self::LARGE_COFFEE_CLIENT_ID];
-      $smallTeaId = $this->idMap[self::SMALL_TEA_CLIENT_ID];
+      $coffeeId = $this->catalogFixtures->idMap[CatalogFixtures::COFFEE_CLIENT_ID];
```

Comment:
Same as with python's PR. This would look much cleaner if you had a getter. You can use magic getters here to provide an api like: `$this->catalogFixtures->coffee_object_id`

http://php.net/manual/en/language.oop5.overloading.php#object.get